.github/workflows/pipeline_building.yml

```diff
@@ -0,0 +1,19 @@
+name: Pipeline building
+
+on: push
+
+jobs:
+  preparation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        run: g++ -std=c++17 -o a.out *.cpp
+      - name: Regaining right
+        run: chmod a+x ./battle_prep.sh
```

Comment:
ezt jo volna, ha nem kellene minden futtataskor megcisnalni. meg amugy lehet bash ize.sh-t is irni.